charts/access/jira/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ include "jira.fullname" . }}
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-type: nlb
+spec:
+  selector:
+    {{- include "jira.selectorLabels" . | nindent 4 }}
+  type: LoadBalancer
+  ports:
+  - name: https
+    port: 443
+    targetPort: 8443
```

Comment:
This should be based off of listenAddress, right?  If that is updated, this will break.

More importantly, do we need to allow the service <-> pod port to be configured at all?  This seems like a knob folks are unlikely to tweak.  Instead, I'd guess customers would want to tweak the inbound port that Jira will hit.